src/main/java/com/gregtechceu/gtceu/data/recipe/generated/WireRecipeHandler.java

```diff
@@ -77,11 +83,54 @@ public static void processWireSingle(TagPrefix wirePrefix, Material material, Wi
                 .save(provider);
 
         WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire")
-                .inputItems(prefix, material)
-                .outputItems(wireGtSingle, material, 2)
-                .duration((int) material.getMass())
+            .inputItems(prefix, material)
+            .circuitMeta(1)
+            .outputItems(wireGtSingle, material, 2)
+            .duration((int) material.getMass())
+            .EUt(getVoltageMultiplier(material))
+            .save(provider);
+
+        WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire_2")
+            .inputItems(prefix, material)
+            .circuitMeta(2)
+            .outputItems(wireGtDouble, material, 1)
+            .duration((int) material.getMass())
+            .EUt(getVoltageMultiplier(material))
+            .save(provider);
+
+        WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire_4")
+            .inputItems(prefix, material, 2)
+            .circuitMeta(4)
+            .outputItems(wireGtQuadruple, material, 1)
+            .duration((int) material.getMass() * 2)
+            .EUt(getVoltageMultiplier(material))
+            .save(provider);
+
+        WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire_8")
+            .inputItems(prefix, material, 4)
+            .circuitMeta(8)
+            .outputItems(wireGtOctal, material, 1)
+            .duration((int) material.getMass() * 4)
+            .EUt(getVoltageMultiplier(material))
+            .save(provider);
+
+        WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire_16")
+            .inputItems(prefix, material, 8)
+            .circuitMeta(16)
+            .outputItems(wireGtHex, material, 1)
+            .duration((int) material.getMass() * 8)
+            .EUt(getVoltageMultiplier(material))
+            .save(provider);
+
+        if (material.hasFlag(GENERATE_FINE_WIRE)) {
+            WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire_fine")
+                .inputItems(prefix, material, 1)
+                .circuitMeta(3)
+                .outputItems(wireFine, material, 8)
+                .duration((int) material.getMass() * 3)
```

Comment:
1.12 had a loop for this yk?

Reply:
would be cleaner :)

Reply:
https://github.com/GregTechCEu/GregTech/blob/master/src/main/java/gregtech/loaders/recipe/handlers/WireRecipeHandler.java#L86-L95